hw/ip/aes/data/aes.hjson

```diff
@@ -133,17 +133,24 @@
     }
   ],
   alert_list: [
-    { name: "ctrl_err_update",
+    //{ name: "informative",
+    //  desc: '''
+    //    The informative alert can currently not be triggered.
+    //    The AES unit recovers from such a condition automatically.
+    //    No further action needs to be taken but this should be monitored by the system.
+    //  '''
+    //}
+    { name: "recoverable",
       desc: '''
-        This minor alert is triggered upon detecting an update error in the Control Register.
-        The AES unit recovers from such a condition automatically.
-        No further action needs to be taken but this should be monitored by the system.
+        The recoverable alert is triggered upon detecting an update error in the Control Register.
+        The content of the Control Register is not modified (See Control Register).
+        The AES unit can be recovered from such a condition by restarting the AES operation, i.e., by successfully updating the Control Register.
+        This should be monitored by the system.
       '''
     }
-    { name: "ctrl_err_storage",
+    { name: "fatal",
```

Comment:
I was actually going to propose a naming scheme that uses prefixes, e.g.:
`fatal_ctrl_err_storage`, `recov_ctrl_err_update`

That way it is still possible to further differentiate the alerts - but we know that they will always start with either of the two prefixes.

WDYT?

Reply:
I think the prefix is a good idea.

Reply:
I like the idea of the prefix but in this particular case I didn't use it for the following reason.

Before this PR, the two alerts were `ctrl_err_update` and `ctrl_err_storage`. There was just one source (the shadow register) that could create alerts. Now with this PR we have multiple sources: shadow register, FSM states, and more will follow (handshake signals, mux selector signals, round counter etc.). Thus I don't think it makes sense to mention any more the alert source in the name but instead just call this the `fatal` alert. Then there is the `recoverable` alert which at the moment can only be triggered by the shadow register as before. I don't consider it useful to have a `fatal` alert and a `recov_ctrl_err_update`. So it's just the `recoverable` alert. Also here we might add more sources soon.

For the guidance the document, my suggestion would be to say something like: "It is recommended to prefix alert names with either `fatal` or `recov/recoverable` to indicate the severeness of the alert and possible implications inside the peripheral (this would be the blue feedback loop in your slide). If a peripheral collects multiple alert sources in just 1 alert per group (`fatal`, `recoverable`) the recommendation is to only use the prefix." In the latter case we can then recommend to add a cause register for providing more information on the actual alert cause, guarded by LC as discussed in another comment.

Side note: For most peripherals I think we want to reduce the number of alerts to a minimum of 2 or 3 to avoid problems like we see for interrupts (see #3421). So maybe the guidance should be: try to expose just one fatal and one recoverable alert. If more alerts are needed, prefix your alerts with either `fatal` or `recov`.

Do you think this is reasonable @msfschaffner ?

Reply:
Your guidance suggestion sounds good to me @vogelpi, I'll try to incorporate this into the documentation (PR will come soon).

Ok to move forward now and amend later (if needed)!

Reply:
Thanks @msfschaffner for the thorough review! I am merging this now.